tests/main/dbus-activation-session-legacy/task.yaml

```diff
@@ -0,0 +1,39 @@
+summary: D-Bus session services support activation without a user of instance systemd
```

Comment:
Is this meant to test how this behaves on Ubuntu 16.04?

Reply:
Yes.  Ubuntu 16.04, CentOS 7, and anything else where the session bus is not managed by systemd.